OSDOCS-3680: Remove backup language from ROSA documentation (#46994)
openshift/openshift-docs: modules/rosa-policy-incident.adoc

There is no backup method available for ROSA clusters with STS.

[id="backup-recovery_{context}"]
== Backup and recovery

> All Red Hat OpenShift Service on AWS cluster metadata from OpenShift Cluster Manager is securely backed up by Red Hat. The following table outlines backup and recovery strategies.

Reviewer: ```suggestion
All Red Hat OpenShift Service on AWS cluster metadata from OpenShift Cluster Manager is securely backed up by Red Hat. Red Hat recommends the following backup strategy:
```

> All Red Hat OpenShift Service on AWS cluster metadata from OpenShift Cluster Manager is securely backed up by Red Hat. The following table outlines backup and recovery strategies.

Reviewer 2: ```suggestion
All Red Hat OpenShift Service on AWS cluster metadata from OpenShift Cluster Manager is securely backed up by Red Hat. The following table outlines backup and recovery strategies:
```

> //Verify if the corresponding tables in rosa-sdpolicy-platform.adoc and policy-incident.adoc also need to be updated. 

Reviewer: The tables should be identical

|Retention
|Notes

.2+|Full object store backup, all cluster persistent volumes (PVs)
|Daily
|7 days
.2+|This is a full backup of all Kubernetes objects like etcd, as well as all PVs in the cluster.

|Weekly
|30 days

|Full object store backup
|Hourly
|24 hour
|This is a full backup of all Kubernetes objects like etcd. No PVs are backed up in this backup schedule.

|Node root volume
|Never
|N/A
|Nodes are considered to be short-term. Nothing critical should be stored on a node's root volume.

|===

* Red Hat does not commit to any Recovery Point Objective (RPO) or Recovery Time Objective (RTO).
* Customers are responsible for taking regular backups of their data
* Customers should deploy multi-AZ clusters with workloads that follow Kubernetes best practices to ensure high availability within a region.

> * In the event an entire cloud region is unavailable, customers must install a new cluster in a different region and restore their apps using their backup data.

Reviewer 2: ```suggestion
* If an entire cloud region is unavailable, customers must install a new cluster in a different region and restore their apps using their backup data.
```
